sdk/rust/oak/src/lib.rs

```diff
@@ -294,6 +345,42 @@ pub fn node_create(
     result_from_status(status as i32, ())
 }
 
+/// Uses the calling node's label-downgrading privilege to creates a new Node running the
+/// configuration identified by `config_name`, running the entrypoint identified by
+/// `entrypoint_name` (for a Web Assembly Node; this parameter is ignored when creating a
+/// pseudo-Node), with the provided `label`, and passing it the given handle.
```

Comment:
The wording seems quite different from that of the channel_create_with_privilege, could you make them a bit more similar? Also I think it may be simpler to maintain here too if the _with_privilege functions just link to the regular functions, and explain only the delta in functionality.

Reply:
Done.